src/test/java/bio/terra/service/filedata/azure/util/ContainerSasTokenSasUrlFactoryTest.java

```diff
@@ -0,0 +1,63 @@
+package bio.terra.service.filedata.azure.util;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.common.category.Unit;
+import com.azure.storage.blob.BlobUrlParts;
+import java.util.Locale;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+@Category(Unit.class)
+public class ContainerSasTokenSasUrlFactoryTest {
+
+  private ContainerSasTokenSasUrlFactory sasUrlFactory;
+  private static final String SIGNATURE_VALUE = "123";
+  private static final String PERMISSIONS_VALUE = "r";
+  private static final String EXPIRY_VALUE = "2015-07-02T08:49Z";
+
+  @Before
+  public void setUp() {
```

Comment:
Our code is inconsistent but I think the form `setup()` is more common than `setUp()`. (Personally, I think it's better to name it `before()` to match the annotation, but that usage has yet to catch on.)
```suggestion
  public void setup() {
```